source/sass/main.scss

```diff
@@ -26,15 +26,32 @@ img {
   max-width: 100%;
 }
 
-ul {
-  list-style: none;
-}
-
 // Utility Classes
 
 // Custom Components
 
-header {
+h1 {
+  font-weight: 700;
+  font-size: 44px;
+  margin: 16px auto 10px;
+  width: 100%;
+}
+
+h2 {
+  font-size: 2rem;
+  font-weight: 700;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  color: $black;
+}
+
```

Comment:
More CSS cleanup for headings can probably be done - I didn't have time to look through them yet.